### PR TITLE
ui: make footer stick to the bottom

### LINKED ISSUE #81: Make sticky footer
The footer doesn't stick to the bottom of the screen if the screen content is too small, so it looks weird:

image

The footer should stick to the bottom in that case and should flow with the content normally if the content of the page is large enough.

See https://css-tricks.com/couple-takes-sticky-footer/

### PR DESCRIPTION
**Pull request by slint**
- Closes #81.

### CODE REVIEW

**Comment by slint** [comment]
Hm, don't merge yet I've across some issues on some pages... I'll update it accordingly

File: invenio_app_rdm/theme/assets/scss/invenio_app_rdm/footer.scss [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
  #footer {
    color: #fff;
+   position: absolute;
```
Comment (fenekku) [inline invenio_app_rdm/theme/assets/scss/invenio_app_rdm/footer.scss]: Hahaha I don't think that's enough ! (I didn't mean for the footer to always be visible at the bottom, but rather to stick it to the bottom if not enough vertical space to make it seem to be sticking to be at the bottom.

I've implemented it in https://github.com/galterlibrary/InvenioRDM-at-NU .

Comment (ppanero) [reply invenio_app_rdm/theme/assets/scss/invenio_app_rdm/footer.scss]: So you mean showing at the bottom always? So even if there is content to scroll, the footer will be visible and then u just scroll the content itself no?

Comment (slint) [reply invenio_app_rdm/theme/assets/scss/invenio_app_rdm/footer.scss]: Hehe, I looked at Zenodo's CSS and thought that was it... 1.5h hour later the ugly truth revealed itself.

I fixed it now though, tried it out on both Zenodo RDM and InvenioRDM and seems to work for all cases/pages.

I think some of the troubles (or at least solving them) came from the fact that some high-level CSS rules coming from Bootstrap and Semantic UI... But there are plans to fix this, so I'm optimistic :)

**Comment by slint** [comment]
Forgot to add, example from working page: https://zenodo-rdm.web.cern.ch/search?q=cole&sort=bestmatch&order=desc&p=1&s=10&l=list

**Comment by fenekku** [comment]
I guess #88 is making this not work on the search page. But we will deal with it later. Otherwise works. Thanks!

**Pull request merged**